lib/capybara/selenium/node.rb

```diff
@@ -134,11 +134,19 @@ def hover
     scroll_if_needed { browser_action.move_to(native).perform }
   end
 
-  def drag_to(element, **)
+  def drag_to(element, **kwargs)
     # Due to W3C spec compliance - The Actions API no longer scrolls to elements when necessary
     # which means Seleniums `drag_and_drop` is now broken - do it manually
     scroll_if_needed { browser_action.click_and_hold(native).perform }
-    element.scroll_if_needed { browser_action.move_to(element.native).release.perform }
+    move_to(element, **kwargs)
+  end
+
+  def move_to(element, modifier_keys: [], **)
```

Comment:
Fair enough.  What's the best way to be able to reuse this code?  The reason I didn't make this private is that the html5 drag plugin calls the external interface of `node`.

Reply:
If it does need to be shared you could just make it private in Selenium::Node, but in this case because Html5Drag is a complete replacement for the standard `drag_to` I would just deal with the small amount of code duplication to keep HtmlDrag self contained and easier to understand.